LCM/dsc/engine/ConfigurationManager/LocalConfigurationManager.c

```diff
@@ -2444,7 +2530,22 @@ void Invoke_SendConfiguration(
     {      
         args->force = MI_TRUE;
     }    
-    Thread_CreateDetached(Invoke_SendConfigurationApply_Internal, NULL, args); 
+
+    {
+	LCMTaskNode * newNode = (LCMTaskNode*)DSC_malloc(sizeof(LCMTaskNode), NitsHere());
```

Comment:
Can this block be encapsulated in a function? It looks like it is identical in the 12 places.